paddle/fluid/framework/ir/mkldnn/cpu_bfloat16_pass.cc

```diff
@@ -199,11 +199,7 @@ class DeQuantizer final : public Quanter {
   bool IsNotPermittedName(const std::string& output_name) const override {
     std::unordered_map<std::string, std::vector<std::string>> block_list{
         {"layer_norm",
-         {"Mean", "Variance"}},        // not used in inference in MKLDNN
-        {"fc", {"ResidualData"}},      // artifical output, already dequantized
-        {"matmul", {"ResidualData"}},  // artifical output, already dequantized
-        {"matmul_v2",
-         {"ResidualData"}}};  // artifical output, already dequantized
```

Comment:
It had to be blocked because otherwise model segfaulted before.
Please run our full Jenkins Pipeline(functional + performance) to check if this change don't break anything.

Reply:
It seems that unit tests already failed because of segfault:
```
2022-11-18 05:52:15 The following tests FAILED:
2022-11-18 05:52:15 	387 - test_analyzer_bert (SEGFAULT)
```

Reply:
It is possible that is because I also removed here ResidualData for FC which has not been merged yet as input. I can bring it back for now and remove it in PR that changes output->input to make it work now.